src/java.base/share/classes/java/util/TimeZone.java

```diff
@@ -130,10 +130,10 @@
  * use is deprecated</strong> because the same abbreviation is often used
  * for multiple time zones (for example, "CST" could be U.S. "Central Standard
  * Time" and "China Standard Time"), and the Java platform can then only
- * recognize one of them. The full list of the deprecated IDs can be viewed
- * at {@link java.time.ZoneId#SHORT_IDS}. It should be noted that not all
- * three-letter time zone IDs are deprecated, as some come from the IANA Time
- * Zone Database (TZDB).
+ * recognize one of them. The full list of the deprecated IDs and their mappings
+ * can be viewed at {@link java.time.ZoneId#SHORT_IDS}. It should be noted that
+ * not all three-letter time zone IDs are deprecated, as some come from the IANA
```

Comment:
This is a bit confusing at least to me as I did not see which SHORT_IDS were not deprecated via the link above, but perhaps I missed it.

Reply:
I see your point, the intent was that there exists three-letter time zone IDs and only the deprecated ones were at `SHORT_IDS`. I have updated the wording to explicitly clarify this.

Reply:
Please disregard the previous message, I talked with Naoto and all the three-letter time zone IDs (and four-letter) are deprecated regardless if they are from the JDK or TZDB, so I will readjust the wording again